Packs/AMP/Integrations/CiscoAMPEventCollector/CiscoAMPEventCollector_description.md

```diff
@@ -0,0 +1,15 @@
+To configure the integration, enter your credentials in the following fields:
+- Client ID
+- API Key
+
+Steps to generate the Credentials
+
+Step 1: Log into the Console, and navigate to Accounts > API Credentials.
+
+Step 2: Click New API Credential to create a new set of Keys.
+
+Step 3: Provide an Application name. Select the Scope of Read-only or Read & Write.
+
+Step 4: Click the Create button. The API Key Details appears. Save this information as some of it will not be available after leaving the screen.
+
+For additional info check [official docs](https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/201121-Overview-of-the-Cisco-AMP-for-Endpoints.html)
```

Comment:
```suggestion
To generate the Credentials:

1. Log into the Console, and navigate to Accounts > API Credentials.

2. Click New API Credential to create a new set of Keys.

3. Provide an Application name. Select the Scope of Read-only or Read & Write.

4. Click the Create button. Save the API Key and Client ID information as some of it will not be available after leaving the screen.

For additional info check [official docs](https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/201121-Overview-of-the-Cisco-AMP-for-Endpoints.html)
```